addons/point_of_sale/static/src/xml/Screens/ClientListScreen/ClientLine.xml

```diff
@@ -6,15 +6,23 @@
             t-on-click="trigger('click-client', {client: props.partner})">
             <td>
                 <t t-esc="props.partner.name" />
+                <span t-if="highlight">
+                    <br/><button class="edit-client-button" t-on-click.stop="showClientDetails">EDIT</button>
```

Comment:
Why ".stop"?

Reply:
Because without the .stop, the event click-client will be triggered instead. 
By adding the .stop, the propagation of the event is removed.